youtube_dl/extractor/podchaser.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+from __future__ import unicode_literals
+import re
+from datetime import datetime
+from ..utils import float_or_none, try_get, str_to_int
+from ..compat import compat_str
+from .common import InfoExtractor
+
+
+class PodchaserIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?podchaser\.com/
+        (?:(?:podcasts)|(?:creators))
+        /[\w\-]+-
+        (?:
+            (?P<creator_id>[\d]+[\w]+)|(?P<podcast_id>[\d]+))
+        (?:/episodes/[\w\-]+-
+            (?P<id>[\d]+))?'''
+
+    _TESTS = [{
+        'url': 'https://www.podchaser.com/podcasts/cum-town-36924/episodes/ep-285-freeze-me-off-104365585',
+        'info_dict': {
+            'id': '104365585',
+            'title': "Ep. 285 – freeze me off",
+            'description': 'cam ahn',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'ext': 'mp3',
+            'categories': ['Comedy'],
+            'series': 'Cum Town',
+            'duration': 3708,
+            'timestamp': 1636560059,
+            'upload_date': '20211110'
+        }
+    }, {
+        'url': 'https://www.podchaser.com/podcasts/the-bone-zone-28853',
+        'info_dict': {
+            'id': '28853',
+            'title': 'The Bone Zone',
+            'description': 'md5:c39acd897170a8bf3ad94fc45dc25060',
+        },
+        'playlist_count': 6
+    }, {
+        'url': 'https://www.podchaser.com/creators/todd-glass-107ZzkFiEQ',
+        'info_dict': {
+            'id': '107ZzkFiEQ',
+            'title': 'Todd Glass',
+            'description': 'md5:f0302588c98d474d4e8551f4914042c9',
+        },
+        'playlist_mincount': 48
+    }, {
+        'url': 'https://www.podchaser.com/podcasts/sean-carrolls-mindscape-scienc-699349/episodes',
+        'info_dict': {
+            'id': '699349',
+            'title': "Sean Carroll's Mindscape: Science, Society, Philosophy, Culture, Arts, and Ideas",
+            'description': 'md5:a1ed5db77378a736b3a85a2331cbc05a'
+        },
+        'playlist_count': 25
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        audio_id, podcast_id, creator_id = compat_str(mobj.group('id')), \
+            compat_str(mobj.group('podcast_id')), compat_str(mobj.group('creator_id'))
+        webpage = self._download_webpage(url, audio_id)
+        page_title = self._html_search_meta(['title', 'og:title', 'twitter:title'], webpage, default=None) \
+            or self._search_regex(
+                r'<h1[^>]*>(.+?)</h1>', webpage, 'title', fatal=False, default="Podchaser Podcast")
```

Comment:
According to the docs, not having a title should be an error, so scrap `fatal=False` in  the second case.